### ISSUE TITLE
Match Paterns don't work after update of icinga-php-thirdparty and icinga-php-library

### ISSUE THREAD

**Issue by ggzengel** [Icinga/icingadb-web#877]
## Describe the bug
I use something like `/services?host.name_ci=px*` and it worked until today where I made an update:
```
Start-Date: 2023-09-22  08:24:02
Install: php-pear:amd64 (1:1.10.13+submodules+notgz+2022032202-2, automatic), php-php-gettext:amd64 (1.0.12-5, automatic)
Upgrade: icinga-php-thirdparty:amd64 (0.11.0-2, 0.12.0-1+debian12), icinga-php-library:amd64 (0.10.1-1, 0.13.0-1+debian12)
End-Date: 2023-09-22  08:24:40
```

Now I have to use like (~) filter expressions.
I couldn't find any change log or documentation about this change.

The only thing I found is:
```
Match Patterns[¶](https://icinga.com/docs/icinga-db-web/latest/doc/04-Security/#match-patterns)
Match patterns allow a very simple form of wildcard matching. Use * in any place to match zero or any characters.
```

## To Reproduce
Upgrade to latest version and call ``/services?host.name_ci=px*`

## Expected behavior
The old behavior or something about the changes.

## Your Environment
```
# uname -a
Linux lx-icingaweb.hq1.local 6.1.0-12-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.52-1 (2023-09-07) x86_64 GNU/Linux

# lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 12 (bookworm)
Release:	12
Codename:	bookworm

# head -n 2 /etc/icingaweb2/enabledModules/*/module.info 
==> /etc/icingaweb2/enabledModules/grafana/module.info <==
Name: Grafana
Version: 2.0.3
==> /etc/icingaweb2/enabledModules/icingadb/module.info <==
Module: icingadb
Version: 1.0.2

# dpkg-query -l | grep icinga
ii  icinga-php-library             0.13.0-1+debian12                       all          Icinga PHP Library - IPL
ii  icinga-php-thirdparty          0.12.0-1+debian12                       all          Icinga 3rd party PHP libraries
ii  icingacli                      2.11.4-2                                all          simple CLI tool for Icingaweb2 and its modules
ii  icingadb-web                   1.0.2-1                                 all          Icinga DB Web for Icinga Web 2
ii  icingaweb2                     2.11.4-2                                all          simple and responsive web interface for Icinga
ii  icingaweb2-common              2.11.4-2                                all          simple and responsive web interface for Icinga - common files
ii  icingaweb2-module-doc          2.11.4-2                                all          simple and responsive web interface for Icinga - documentation module
ii  icingaweb2-module-monitoring   2.11.4-2                                all          simple and responsive web interface for Icinga - monitoring module
ii  php-icinga                     2.11.4-2                                all          PHP library to communicate with and use Icinga
```

**Comment by yhabteab** [comment]
Hi, thanks for reporting.

> I couldn't find any change log or documentation about this change.

Here is a release note about this https://github.com/Icinga/ipl-web/releases/tag/v0.9.0, though it may not work properly without the latest changes in Icinga DB Web. However, Icinga DB Web will receive the updates next week as well.

**Comment by nilmerg** [comment]
Using the new operators `~` and `!~` is fully working with Icinga DB Web v1.0.x. The note about this change will be part of the release announcement of Icinga DB Web v1.1. Though, it would have certainly cleared things up to also have included it in the Icinga Web v2.12 announcement :neutral_face:

**Comment by ggzengel** [comment]
Thank you.
